components/filemanager/data-binding/overview.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Overview
+page_title: FileManager - Data Binding Overview
+description: Data Binding basics in the FileManager for Blazor.
+slug: filemanager-data-binding-overview
+tags: telerik,blazor,treelist,data,bind,databind,databinding,basics
+published: True
+position: 0
+---
+
+# FileManager Data Binding Basics
+
+This article explains the different ways to provide data to a FileManager component and the properties related to data binding. Reviewing this article will explain the basics of how you can describe the hierarchy of items in your data source to the treelist component so they can render.
+
+@[template](/_contentTemplates/common/general-info.md#valuebind-vs-databind-link)
+
+First, review:
+
+* The available (bindable) [features of a FileManager items](#fileManager-item-features).
+* How to match fields in the model with the FileManager item [data bindings](#data-bindings).
+
+There are two modes of providing data to a FileManager, and they both use the items' features. Once you are familiar with the current article, choose the data binding more you wish to use:
+
+* [Flat data]({%slug filemanager-data-binding-flat-data%}) - a collection of self-referencing items with parent-child relationships See the `Id` and `ParentId` settings.
+
+* [Hierarchical data]({%slug filemanager-data-binding-hierarchical-data%}) - each item holds its children in a nested property. See the `Directories` setting.
+
+## FileManager Item Features
+
+The FileManager has features that map to properties in the model. The following model uses property names that will work automatically, with no additional FileManager configuration:
+
+````CSHTML
+````
+
+The above model properties have the following meaning for the FileManager:
+
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Property | Description |
+|----------|----------|----------|
+| **Item features** | |
+| `Name` | The name of the file.
+| `Size` | The size of the file.
+| `Path` | The path of the file.
```

Comment:
This might be OK in the deployed version of the docs, but it is displayed like this for now:
![image](https://user-images.githubusercontent.com/1857705/167377279-6f7a3b98-877a-48a7-9bdf-6517b90bf579.png)

Can we re-check if the current appearance is expected?